kubernetes/helm/ckeditor-pdf-converter/.helmignore

```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
+# This supports shell glob matching, relative path matching, and
```

Comment:
Again, is "glob" legit?

Reply:
https://github.com/cksource/ckeditor-cs-on-premises-infrastructure/pull/15#discussion_r1042179297